Library/Formula/leiningen.rb

```diff
@@ -53,5 +51,4 @@ def caveats; <<-EOS.undent
     EOS
     system "#{bin}/lein", "test"
```

Comment:
test is failing: http://bot.brew.sh/job/Homebrew%20Pull%20Requests/20480/version=yosemite/testReport/junit/brew-test-bot/yosemite/test_leiningen/